flash_tutorials/dravidian_languages_question_answering/.meta.yml

```
@@ -0,0 +1,18 @@
+title: Question Answering for Dravidian Languages
+author: Karthik Rangasai Sivaraman (karthikrangasai@gmail.com)
+created: 2021-12-15
+updated: 2021-12-15
+license: CC BY-SA
+build: 3
+tags:
+  - Text
+  - Question Answering
+description: |
+  This tutorial covers using Lightning Flash and it's integration with Hugging Face Transformers to train a Transformer
+  model (XLM-RoBERTa) on SQuAD type dataset for the dravidian languages. We show how easy it is to use a Hugging Face
+  Transformers model with the all goodness provided by PyTorch Lightning using Flash.
+requirements:
+  - lightning-flash[text]>=0.5.2
+accelerator:
+  - GPU
+  - CPU
```

```suggestion
  - CPU
datasets:
  web:
    - https://pl-flash-data.s3.amazonaws.com/kaggle_chaii.zip
```
or rather use the Kaggle version directly unless you made some preprocessing...
BUT if you did some pre-processing it all be also mentioned and well described in the notebook :]

---

I have added the kaggle dataset in meta file. I have not done any preprocessing, so directly pointing to the kaggle dataset.